src/cmds/testing/input/input_test/Mybuild

```diff
@@ -1,5 +1,4 @@
-
-package embox.cmd
+package embox.cmd.testing.input
```

Comment:
Please change package name for templates where `input_test` is included.
```
> grep input_test -rn ./templates/
./templates/arm/ti816x/mods.config:51:	include embox.cmd.input_test
./templates/arm/ti816x-qt/mods.config:44:	include embox.cmd.input_test
```

Reply:
Thank you, fixed in https://github.com/embox/embox/pull/1950